Lesson19/irem/script.js

```diff
@@ -0,0 +1,262 @@
+/*
+===========================================================
+  SHOPPING CART APPLICATION
+===========================================================
+In this project, you'll create a simple Shopping Cart to
+simulate adding items, removing items, calculating totals,
+and applying discounts.
+
+You'll practice:
+1. Classes and objects
+2. Encapsulation and abstraction
+3. Methods (functions inside a class)
+4. Arrays and basic array methods (push, filter, find)
+5. Conditional statements (if-else)
+
+Below is a step-by-step guide with comments explaining
+each part. You can test each step by running the code in
+Node.js or a browser console.
+*/
+
+/*
+-----------------------------------------------------------
+  STEP 1: Create the ShoppingCart Class
+-----------------------------------------------------------
+1. Define a `ShoppingCart` class.
+2. Add a constructor that initializes an empty private 
+   array `_items` to store the cart items.
+3. Add a `viewCart` method to display all items in the cart.
+*/
+/* 
+Price:
+{
+  amount: 10.99,
+  currency: 'EUR'
+}
+
+*/
+
+const itemsInTheStore = {
```

Comment:
Rename this priceList or similar, so it's easier to understand what is inside this object.